control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
```

Comment:
Yes. It replaces that entirely. The reason I did this is so we have one repo to clone (this one) and then one command to run `. commcarehq-ansible/control/init.sh` to bootstrap a dev account.